packages/manager/cypress/e2e/core/vpc/vpc-linodes-update.spec.ts

```diff
@@ -0,0 +1,302 @@
+/**
+ * @file Integration tests for VPC assign/unassign Linodes flows.
+ */
+
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import {
+  mockGetSubnets,
+  mockCreateSubnet,
+  mockGetVPC,
+  mockGetVPCs,
+} from 'support/intercepts/vpc';
+import {
+  subnetFactory,
+  vpcFactory,
+  linodeFactory,
+  linodeConfigFactory,
+} from '@src/factories';
+import { ui } from 'support/ui';
+import { randomNumber, randomLabel } from 'support/util/random';
+import { mockGetLinodes } from 'support/intercepts/linodes';
+import {
+  mockCreateLinodeConfigInterfaces,
+  mockGetLinodeConfigs,
+} from 'support/intercepts/configs';
+import {
+  vpcAssignLinodeRebootNotice,
+  vpcUnassignLinodeRebootNotice,
+} from 'support/constants/vpc';
+import { VPC, Linode, Config } from '@linode/api-v4/types';
+
+describe('VPC assign/unassign flows', () => {
+  let mockVPCs: VPC[];
+  let mockLinode: Linode;
+  let mockConfig: Config;
+
+  before(() => {
+    mockVPCs = vpcFactory.buildList(5);
+
+    mockLinode = linodeFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+    });
+
+    mockConfig = linodeConfigFactory.build({
+      id: randomNumber(),
+    });
+  });
+
+  /*
+   * - Confirms that can assign a Linode to the VPC when feature is enabled.
+   */
+  it.skip('can assign Linode(s) to a VPC', () => {
```

Comment:
Let's not forget to unskip this!
```suggestion
  it('can assign Linode(s) to a VPC', () => {
```